uk.co.compucorp.civicrm.hrcore/gulp/tasks/requirejs.js

```diff
@@ -173,6 +195,7 @@ function requireJsTask (cb) {
       sequence.push(utils.spawnTaskForExtension('requirejs:dependencies', extensionDependenciesTask));
     }
 
+    sequence.push(utils.spawnTaskForExtension('requirejs:sourceMapPath', requireJsSourceMapPath));
```

Comment:
fixing the source map url shouldn't be a public task, it should be something done as part of the main task